nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -1004,18 +1008,23 @@ define([
         function updateState(newUiState) {
             if (newUiState && newUiState in States) {
                 let newTab;
-                model.setItem('state.state', newUiState);
-                const stateDiff = JSON.parse(JSON.stringify(States[newUiState].ui));
-                stateDiff.selectedFileType = state.selectedFileType;
-                // update selections
-                if (stateDiff.tab.tabs[state.tab.selected].enabled) {
-                    stateDiff.tab.selected = state.tab.selected;
-                } else {
-                    newTab = stateDiff.defaultTab;
-                }
-                state = stateDiff;
-                if (newTab) {
-                    switchToTab(newTab);
+                const oldState = model.getItem('state.state');
+                if (oldState !== newUiState) {
+                    model.setItem('state.state', newUiState);
+                    const stateDiff = JSON.parse(JSON.stringify(States[newUiState].ui));
+                    stateDiff.selectedFileType = state.selectedFileType;
+                    // update selections
+                    if (state.tab.selected) {
```

Comment:
Fixes an error caused by the assumption that `state.tab.selected` would always be defined; if there is no tab open, it will be null.